ui/e2e/src/pages/AppHomePage.ts

```diff
@@ -52,6 +52,7 @@ export class AppHomePage {
   async clickDashboardItem(dashboardName: string) {
     const dashboardButton = this.page.getByRole('button', {
       name: dashboardName,
+      exact: true,
```

Comment:
Needed because there are multiple dashboards with the name "panels" in them now.